code/modules/mob/mob_movement.dm

```diff
@@ -228,16 +228,16 @@
 
 				if(GRAB_AGGRESSIVE)
 					move_delay = world.time + 10
-					if(!prob(25))
+					if(!prob(75))
 						return 1
-					mob.visible_message("<span class='warning'>[mob] has broken free of [G.assailant]'s grip!</span>")
+					mob.visible_message("<span class='danger'>[mob] has broken free of [G.assailant]'s grip!</span>")
 					qdel(G)
 
 				if(GRAB_NECK)
 					move_delay = world.time + 10
-					if(!prob(5))
+					if(!prob(50))

```

Comment:
If we're going from "Inescapable permastun" to "Inescapable pemastun as long as the choker walks back and forth while choking" this is a step backwards